ci/unit/test_post_always.sh

```diff
@@ -26,6 +26,9 @@ ssh "$SSH_KEY_ARGS" jenkins@"$NODE" \
    $(cat "$mydir/test_post_always_node.sh")"
 
 case $STAGE_NAME in
+    *Bullseye bdev*)
```

Comment:
(lint) Couldn't parse this case item. Fix to allow more checks. [SC1073]
(lint) Expected ) to open a new case item. Fix any mentioned problems and try again. [SC1072]
(lint) Did you forget to move the ;; after extending this case item? [SC1085]